ubuntudesign/www.ubuntu.com
# Promote training

## Done

* moved training from under OpenStack to under cloud
* updated/fixed the nav partials to accommodate the changes

## QA

[demo site](http://www.ubuntu.com-promote-training.demo.haus/cloud/training)

1. go to /cloud/training and all the contact us and thank you pages and see that the links work, they look ok and the nav is correct
2. go to /cloud/openstack/management/contact-us and thank-you and see that the nav looks ok

## Issue / Card

[trello card](https://trello.com/c/njvrGYxV)
